### PR TITLE
Adding azure-mgmt-monitor

### PR DESCRIPTION
**Pull request by tim-werner**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/azure-mgmt-monitor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
```
Comment (marcelotrevisani) [inline recipes/azure-mgmt-monitor/meta.yaml:5]:
```suggestion
```

File: recipes/azure-mgmt-monitor/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - msrestazure >=0.4.32,<2.0.0
+    - azure-common ~=1.1
+
+test:
+  imports:
+    - azure
+    - azure.mgmt
+    - azure.mgmt.monitor
+  commands:
+    - python -m unittest discover -p "test_*.py"
```
Comment (marcelotrevisani) [inline recipes/azure-mgmt-monitor/meta.yaml:38]:
```suggestion
```

File: recipes/azure-mgmt-monitor/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+requirements:
+  host:
+    - python >3
+    - pip
+  run:
+    - python >3
+    - msrest >=0.5.0
+    - msrestazure >=0.4.32,<2.0.0
+    - azure-common ~=1.1
```
Comment (marcelotrevisani) [inline recipes/azure-mgmt-monitor/meta.yaml:30]:
```suggestion
    - azure-common >=1.1,<2.0
```

File: recipes/azure-mgmt-monitor/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >3
+    - pip
+  run:
+    - python >3
```
Comment (marcelotrevisani) [inline recipes/azure-mgmt-monitor/meta.yaml:27]:
```suggestion
    - python >3.0
```

File: recipes/azure-mgmt-monitor/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  sha256: 68e9e1305f2792f865188de916c6ab987af9316ae5e9ec612c1bf28f43f6129c
+  
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >3
```
Comment (marcelotrevisani) [inline recipes/azure-mgmt-monitor/meta.yaml:24]:
```suggestion
    - python >3.0
```

**Comment by tim-werner** [comment]
Thanks for the review

**Pull request merged**